src/components/pages/observation/list/views/stats/totals.tsx

```diff
@@ -14,7 +16,12 @@ const Totals = ({ filter, observationData, speciesGroup }) => {
 
     const filteredGroups = speciesGroup.filter((sg) => filter.sGroup.includes(sg.id));
 
-    const filteredCounts = filteredGroups.map((v) => observationData.ag.groupSpeciesName[v.name]);
+    const filteredCounts = filteredGroups.map((v) =>
+      observationData.ag.groupSpeciesName[v.name] != undefined
```

Comment:
@prakhar-s `!= undefined` can be removed if you are just doing defined check since it's redundant also please use `===` or `!==` check if using it anywhere in future

Reply:
or even better you can just do `    const filteredCounts = filteredGroups.map((v) => observationData.ag.groupSpeciesName[v.name] || 0);	`